addons/web/static/tests/views/fields/datetime_field.test.js

```diff
@@ -63,13 +63,18 @@ test("DatetimeField in form view", async () => {
         type: "form",
         resModel: "partner",
         resId: 1,
-        arch: '<form><field name="datetime"/></form>',
+        arch: `<form>
+            <field name="datetime"/>
+            <field name="datetime" readonly="1"/>
+        </form>`,
     });
 
     const expectedDateString = "02/08/2017 12:00:00"; // 10:00:00 without timezone
     expect(".o_field_datetime button").toHaveValue(expectedDateString, {
         message: "the datetime should be correctly displayed",
```

Comment:
not related to this commit, but this assertion doesn't make sense. We should check the innerText instead, or in addition if it's somehow relevant to assert the value attribute (not even sure why/if it is useful).